setup.py

```diff
@@ -61,7 +61,7 @@ def open_local(paths, mode="r", encoding="utf8"):
         "Build fast. Run fast."
     ),
     "long_description": long_description,
-    "packages": find_packages(),
+    "packages": find_packages(exclude=("tests", "tests.*")),
```

Comment:
None of the other dirs get added, correct?

Reply:
Yeah, for now only `tests` and `sanic` are included in wheel distribution. This PR removes `tests` and leaving `sanic` untouched.